src/utils/spo.ts

```diff
@@ -671,5 +673,49 @@ export const spo = {
 
     const customActionOnSite = await getById(webUrl, id, "Site");
     return customActionOnSite;
+  },
+
+  async getWebRetentionLabelInformationByName(webUrl: string, name: string): Promise<ListItemRetentionLabel> {
```

Comment:
Hi @nicodecleyre, this refactoring (both functions) is not part of the issue specs, could you return this to the original state and just implement the assetId part?
Refactoring to util functions should be discussed beforehand so we agree on if it's useful and the best way to do it. 

To take this function as an example: it is a bit problematic in the naming and in what it returns. It's named something with 'web', but it returns a 'ListItemRetentionLabel' object, which is in fact a RequestObject for the setComplianceTag endpoint. That does not really make sense.

It would be more logical to return the 'SiteRetentionLabel' and do the mapping in the command. But that would make this function so small its usefulness is limited.

Aside from that the SetComplianceTag endpoint will be deprecated in the near future and I was planning on creating an issue to refactor these occurrences to another endpoint, which does not need the same properties and the retrieving of labels by name will be even less useful then.

Concluding: please return this to its original state and focus on just the issue specs. The next time you think a util function would be handy, write a comment so we can discuss it.

Reply:
Just one extra comment: I understand you did this to remove the `executeCommandWithOutput` call, but let's just replace that with the http request you are doing here. It's essentially just one request, so perfectly fine to swap it.

Reply:
ok 😭

Reply:
😂 ik heb met je te doen. Je was er vast blij mee... maar dat is de reden waarom vooraf checken wel handig is

Reply:
Ik klink wat streng wellicht, zo is het niet bedoeld 😇